xep-0198.xml

```diff
@@ -388,7 +399,12 @@
 </failed>
 ]]></example>
   <p>In both of these failure cases, the server SHOULD allow the client to bind a resource at this point rather than forcing the client to restart the stream negotiation process and re-authenticate.</p>
-  <p>If the former stream is resumed and the server still has the stream for the previously-identified session open at this time, the old stream SHOULD be terminated.</p>
+  <p>If the former stream is resumed and the server still has the stream for the previously-identified session open at this time, the server SHOULD send a 'conflict' stream error and close that stream</p>
+<example caption='Server returns error if client attempts to enable stream management before resource binding'><![CDATA[
```

Comment:
I think this could use a paragraph of text, otherwise the example kinda comes out of nowwhere

Reply:
I actually made a minor change in the line preceding the example. I feel that more text to illustrate the error puts the thing out of proportion.